spec.bs

```diff
@@ -157,7 +197,11 @@ When documents move into and out of <a href="https://webkit.org/blog/427/webkit-
 
 ### HTML: <a href="https://html.spec.whatwg.org/multipage/webappapis.html#event-loop-processing-model">HTML Event Loop Processing Model</a> ### {#html-event-loop}
 
-Step #1 currently ignores "tasks whose associated {{Document}}s are not [=fully active=]". Modify this clause to also ignore [=task/freezable=] tasks whose associated {{Document}} is in the [=/frozen=] state.
+Step #1 currently ignores "tasks whose associated {{Document}}s are not [=fully active=]". Modify this clause to also ignore [=task source/freezable=] tasks whose associated {{Document}} is in the [=/frozen=] state.
+
+After Step #11 during the [=Update the rendering=] add the following step.
+
+For each [=fully active=] Document in docs, if visibility in viewport or frame rendered states have changed queue a task run the [=update frame throttling steps=].
```

Comment:
```suggestion
For each [=fully active=] {{Document}} in |docs|, if visibility in viewport or frame rendered states have changed, then [=queue a task=] to run the [=update frame throttling steps=].
```

Reply:
In addition to the minor corrections here, I don't know what "visibility in viewport" or "frame rendered states" means, similar to the above.